src/StringUtils.hs

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
While I recognize that unescaping the string is important here, I’m not sure I like having another Parsec parser just for that. Maybe we can think of a more lightweight way of doing this?

Reply:
I actually spent a while trying to figure out something but wasn't able to find anything unfortunately. The options I found used external packages not included in Carp, and I didn't want to add a package on my own. 

For example, [Text.Replace](http://hackage.haskell.org/package/text-replace-0.0.0.6/docs/Text-Replace.html) could do the work.

Reply:
Actually I dont think it is required since rawstrings are merged now.